bin/cli.js

```diff
@@ -48,7 +47,6 @@ const cliFlags = yargs
   })
   .option('config', {
     'describe': 'Path to json config file',
-    'coerce': (arg) => getConfigFromFile(arg),
```

Comment:
This feels weird, in that cliFlags contains some preprocessed values. So I moved it to later to do it explicitly.